lib/disability_compensation/providers/rated_disabilities/lighthouse_rated_disabilities_provider.rb

```diff
@@ -37,14 +38,18 @@ def transform(data)
           rated_disability_id: 0,
           rating_decision_id: 0,
           rating_percentage: rated_disability['rating_percentage'],
+          # TODO: figure out if this is important
           related_disability_date: DateTime.now
         )
       end
     DisabilityCompensation::ApiProvider::RatedDisabilitiesResponse.new(rated_disabilities:)
   end
 
   def decision_code_transform(decision_code_text)
-    if decision_code_text&.downcase == 'Service Connected'.downcase
+    service_connected = decision_code_text&.downcase == 'Service Connected'.downcase ||
+                        decision_code_text&.downcase == '1151 Granted'.downcase
```

Comment:
Lighthouse is sending back strings from their source for the "decision code", which affects the front-end as it is looking for a "code" rather than simply a string.

"1151 Granted" has the same treatment as "Service Connected".

They are looking into if there are any other values like that.